os/include/tinyara/arch.h

```diff
@@ -445,6 +445,16 @@ void up_allocate_secure_context(TZ_ModuleId_t size);
  ****************************************************************************/
 
 void up_free_secure_context();
```

Comment:
Could you add `void` at argument with different commit? It's the TizenRT coding rule.

Reply:
Done.. Thanks.